Makefile

```diff
@@ -88,32 +89,54 @@ jsonnet/example/manifests: jsonnet/example/main.jsonnet $(JSONNET) $(GOJSONTOYAM
 	$(JSONNET) -m jsonnet/example/manifests jsonnet/example/main.jsonnet | xargs -I{} sh -c 'cat {} | $(GOJSONTOYAML) > {}.yaml' -- {}
 	find jsonnet/example/manifests -type f ! -name '*.yaml' -delete
 
-.PHONY: container
-container: Dockerfile
-	@docker build --build-arg BUILD_DATE="$(BUILD_TIMESTAMP)" \
+.PHONY: container-dev
+container-dev:
+	@docker build \
+	    --build-arg BUILD_DATE="$(BUILD_TIMESTAMP)" \
 		--build-arg VERSION="$(VERSION)" \
 		--build-arg VCS_REF="$(VCS_REF)" \
 		--build-arg VCS_BRANCH="$(VCS_BRANCH)" \
 		--build-arg DOCKERFILE_PATH="/Dockerfile" \
-		-t $(DOCKER_REPO):$(VCS_BRANCH)-$(BUILD_DATE)-$(VERSION) .
-	@docker tag $(DOCKER_REPO):$(VCS_BRANCH)-$(BUILD_DATE)-$(VERSION) $(DOCKER_REPO):latest
-
-.PHONY: container-push
-container-push: container
-	docker push $(DOCKER_REPO):$(VCS_BRANCH)-$(BUILD_DATE)-$(VERSION)
-	docker push $(DOCKER_REPO):latest
+		-t $(DOCKER_REPO):$(VCS_BRANCH)-$(BUILD_DATE)-$(VERSION) \
+		.
+	docker tag $(DOCKER_REPO):$(VCS_BRANCH)-$(BUILD_DATE)-$(VERSION) $(DOCKER_REPO):latest
+
+.PHONY: container-nightly-push
+container-nightly-push:
```

Comment:
Does this job care if it's nightly? 🤔

Reply:
```suggestion
container-build-push:
```

Reply:
to mention we are also building now

Reply:
Agreed, we should name it according to what it does rather than why we might use it